lib/elixir/src/elixir_rewrite.erl

```diff
@@ -20,44 +21,42 @@
 -define(string_chars, 'Elixir.String.Chars').
 -define(system, 'Elixir.System').
 -define(tuple, 'Elixir.Tuple').
--define(function, 'Elixir.Function').
 
 %% Inline
 
 %% Inline rules are straightforward, they keep the same
 %% number and order of arguments and show up on captures.
 
 inline(?atom, to_charlist, 1) -> {erlang, atom_to_list};
-inline(?io, iodata_length, 1) -> {erlang, iolist_size};
-inline(?io, iodata_to_binary, 1) -> {erlang, iolist_to_binary};
-inline(?integer, to_string, 1) -> {erlang, integer_to_binary};
-inline(?integer, to_string, 2) -> {erlang, integer_to_binary};
+
+inline(?function, capture, 3) -> {erlang, make_fun};
+inline(?function, info, 1) -> {erlang, fun_info};
+inline(?function, info, 2) -> {erlang, fun_info};
+
 inline(?integer, to_charlist, 1) -> {erlang, integer_to_list};
 inline(?integer, to_charlist, 2) -> {erlang, integer_to_list};
-inline(?list, to_atom, 1) -> {erlang, list_to_atom};
-inline(?list, to_existing_atom, 1) -> {erlang, list_to_existing_atom};
-inline(?list, to_float, 1) -> {erlang, list_to_float};
-inline(?list, to_integer, 1) -> {erlang, list_to_integer};
-inline(?list, to_integer, 2) -> {erlang, list_to_integer};
-inline(?list, to_tuple, 1) -> {erlang, list_to_tuple};
+inline(?integer, to_string, 1) -> {erlang, integer_to_binary};
+inline(?integer, to_string, 2) -> {erlang, integer_to_binary};
 
-inline(?kernel, '+', 2) -> {erlang, '+'};
-inline(?kernel, '-', 2) -> {erlang, '-'};
-inline(?kernel, '+', 1) -> {erlang, '+'};
-inline(?kernel, '-', 1) -> {erlang, '-'};
+inline(?io, iodata_length, 1) -> {erlang, iolist_size};
+inline(?io, iodata_to_binary, 1) -> {erlang, iolist_to_binary};
+
+inline(?kernel, '!=', 2) -> {erlang, '/='};
+inline(?kernel, '!==', 2) -> {erlang, '=/='};
 inline(?kernel, '*', 2) -> {erlang, '*'};
-inline(?kernel, '/', 2) -> {erlang, '/'};
+inline(?kernel, '+', 1) -> {erlang, '+'};
+inline(?kernel, '+', 2) -> {erlang, '+'};
 inline(?kernel, '++', 2) -> {erlang, '++'};
+inline(?kernel, '-', 1) -> {erlang, '-'};
+inline(?kernel, '-', 2) -> {erlang, '-'};
 inline(?kernel, '--', 2) -> {erlang, '--'};
-inline(?kernel, 'not', 1) -> {erlang, 'not'};
+inline(?kernel, '/', 2) -> {erlang, '/'};
```

Comment:
I don't expect a new entry to be in the right place according to the text editor ordering (which probably changes depending on the editor you are using), but I had to sort it somehow. if you look at it you can see an order which before there was not. Maybe a few lines could have left unmoved to avoid changes in git history, that just slipped through